### PR TITLE
Better documentation and simplified directory structure

### PR DESCRIPTION
**Pull request by nathanchapman**

### CODE REVIEW

File: backend/transactions-service/src/migrations.ts [hunk 1]
```diff
@@ -6,16 +6,20 @@ import { Database } from 'better-sqlite3';
...
-    await database.exec(`
+    database.exec(`
         CREATE TABLE IF NOT EXISTS transactions
-        (id TEXT PRIMARY KEY, date TEXT NOT NULL)
+        (
+            id TEXT PRIMARY KEY NOT NULL,
+            date TEXT NOT NULL
+        );
     `);
+    // console.log(database.pragma('table_info(transactions)'));
```
Comment (nathanchapman) [inline backend/transactions-service/src/migrations.ts:16]: Intentionally added this to aid in debugging

**Review by trhodeos — APPROVED** [review]
RIP multiple top dogs

**Comment by nathanchapman** [comment]
@trhodeos LOL
`top-dogs` -> `top-dog` to match README project name
`transaction-service` -> `transactions-service` to match TypeScript interface and class

**Review by djtaylor8 — APPROVED** [review]
🤌

**Pull request merged**